src/components/@shared/VerifiedBadge/index.tsx

```diff
@@ -22,11 +25,25 @@ export function Badge({
       className={cx({
         mainLabel: true,
         isValid,
+        isWarning: matchVerifiable === false,
         [className]: className
       })}
     >
+      {matchVerifiable === false && (
+        <Tooltip content="gx:dependsOn could not be found" />
```

Comment:
issue(blocking): we need to change the text to something that is better communicating why the id could not be matched
`dependsOn` is not a necessary field, so it is possible that it is missing.

Reply:
comment: you can use some dummy texts for now for the different tooltips